.github/workflows/test-checks.yaml

```
@@ -183,13 +183,60 @@ jobs:
+        compiler_profile: [clang-libstdcxx]
+        include:
+          - compiler_profile: clang-libstdcxx
+            check_name: x86_64-linux-clang-libstdc
+    name: run-address-sanitizer-e2e-release (${{ matrix.check_name }})
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v6
+
+      - name: Export GitHub Actions cache variables
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)